Las/Utils/Filetype.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Lucidtech.Las.Utils
+{
+    /// <summary>
+    /// Help determine the type of a file, inspired by pythons <c>imghdr.what()</c>.
+    /// </summary>
+    public static class FileType
+    {
+        /// <summary>
+        /// Tests the type of file.
+        /// </summary>
+        /// <param name="fileName"> the name of the file that is to be classified </param>
+        /// <returns> the name of the file type that matches, or an empty string if the file does not match</returns>
+        public static string WhatFile(string fileName)
+        {
+            var fileTestList = new List<Func<byte[],string>>()
+            {
+                TestPdf, 
+                TestJpeg
+            } ; 
```

Comment:
Should whitespace between "}" and ";" be removed?

Reply:
yes